src/Workspaces/Core/Portable/FindSymbols/SymbolAndProjectId.cs

```diff
@@ -0,0 +1,175 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis.Shared.Utilities;
+
+namespace Microsoft.CodeAnalysis.FindSymbols
+{
+    /// <summary>
+    /// Represents a symbol and the project it was acquired from.
+    /// It should always be the case that if you have the original solution
+    /// that this symbol came from, that you'll be able to find this symbol
+    /// in the compilation for the specified project.
+    /// </summary>
```

Comment:
I suggest calling out that the "same" symbol can be acquired from multiple different projects, and providing an example of how that might happen.